packages/perseus/src/widgets/interactive-graphs/graphs/circle.tsx

```diff
@@ -17,17 +24,10 @@ export function CircleGraph(props: CircleGraphProps) {
 
     return (
         <>
-            <Circle
+            <MovableCircle
                 center={center}
                 radius={getRadius(graphState)}
-                fillOpacity={0}
-                color={color.blue}
-            />
-            <StyledMovablePoint
-                point={center}
-                onMove={(newCenter) => {
-                    dispatch(moveCenter(newCenter));
-                }}
+                onMove={(newCenter) => dispatch(moveCenter(newCenter))}
```

Comment:
Question: Are we going to add the fill color on hover? I think you mentioned it wasn't added yet, but will that be a separate PR?

Reply:
That's coming in https://khanacademy.atlassian.net/browse/LEMS-1978